src/applications/vaos/appointment-list/redux/actions.js

```diff
@@ -220,40 +288,8 @@ export function fetchFutureAppointments({ includeRequests = true } = {}) {
       ];
 
       if (includeRequests) {
-        promises.push(
-          getAppointmentRequests({
```

Comment:
Calling 'getAppointmentRequests' directly doesn't not make the additional call for facility data.